requirements.txt

```diff
@@ -16,3 +15,4 @@ wandb==0.15.10
 tenacity
 antlr4-python3-runtime==4.11
 wikipedia
+mathgenerator @ git+https://github.com/bkb2135/mathgenerator.git@main
```

Comment:
let's not forget to change this when https://github.com/synapse-alpha/mathgenerator/pull/1#pullrequestreview-1855272511 is merged